guide/.ballerina/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Shall we not commit the .ballerina folder?

Reply:
Okay. I will update it